src/styles/blocks/card.css

```diff
@@ -0,0 +1,104 @@
+.card {
+  width: 200px;
```

Comment:
Your card width is 202px, but must be 200px, please add box-sizing: border-box to fix it
![image](https://user-images.githubusercontent.com/54936044/224479299-91424e7c-fc82-4db3-8124-eb63e162579b.png)